definitions/checks/pulpcore/not_running.rb

```diff
@@ -0,0 +1,21 @@
+module Checks::Pulpcore
+  class NotRunning < ForemanMaintain::Check
+    metadata do
+      for_feature :pulpcore
+      description 'Check for running pulpcore tasks'
+      tags :pre_upgrade
```

Comment:
Are we still using `pre_upgrade` tags, or should we add this "manually" to the upgrade process?

Reply:
Right now we are -- however, I still want to go this route https://github.com/theforeman/foreman_maintain/pull/763 and therefore explicit would be better?